docs/src/4.10.Text.mdx

```diff
@@ -6,13 +6,22 @@ import {Text} from './jsx/allLiveEditors'
 
 ## Props
 
-| Name       | Type     | Default | Description                     |
-| ---------- | -------- | ------- | ------------------------------- |
-| `children` | `Text[]` | `[]`    | array of Text markers to render |
+| Name            | optional | Type      | Default | Description                            |
+| --------------- | -------- | --------- | ------- |
+| `children`      |          | `Text[]`  | `[]`    | array of Text markers to render        |
+| `enableBgColor` | yes      | `boolean` |         | show solid background color if enabled |
```

Comment:
I would probably call this the full words: `enableBackgroundColor`, though maybe we could change the API slightly....

If the marker has a colors array then there is no need to set `enableBackgroundColor` - we just always apply the first color in the `colors` array as the text color and the second as the background color.  Then we could change this prop to be called `autoColorBackground` and if it's set to true _and_ the marker has has a single `color` then we set the background color.  That way one could use a mix of markers with 2 colors (text & background) and others w/ only a single color and the single color markers would still have a transparent background as they do now.

Thoughts?

Reply:
I love the idea! only I'd call it `autoBackgroundColor`:) Will update!